2017/11/29/systemd-primer-from-the-trenches.html.md

```diff
@@ -0,0 +1,514 @@
+---
+author: Ian Neilsen
+title: "systemd: A primer from the trench's"
+tags: hosting, systemd, systemctl 
+---
+
+# systemctl - lets get back to basics
+
+''help me systemd you are my only hope''. If we start back at the beginning, sometimes going back to day dot
+often brings clarity to what seems like hopeless or frustrating situation for users from SysV world. Caveat, 
+I previously worked at Red Hat for many years before joining the excellent team at End Point and have been using systemd for as long. 
+I quite honestly have forgotten most of the SysV or init days. Although at End Point we work daily on Debian, Ubuntu, Centos and BSD variants.
+
+So here is a short and sweet primer to get your fingers wet, before we dive into some of the heavier subjects with systemd.
+
+Did you know that systemd has many utilities it can run:
+
+* systemctl
+* timedatectl
+* journalctl
+* loginctl
+* notify
```

Comment:
These non-*ctl utilities all need `systemd-` in front, and to be lowercase and "nspawn" instead of "N spawn".

Reply:
will adjust where necessary

Reply:
prefixed systemd